tests/test_heap.py

```diff
@@ -93,16 +94,16 @@ def setup_heap(start_binary, bug_no):
 
 def test_try_free_invalid_overflow(start_binary):
     chunks = setup_heap(start_binary, 1)
-    
+
     result = gdb.execute('try_free {}'.format(hex(chunks['a'])), to_string=True)
     assert 'free(): invalid pointer -> &chunk + chunk->size > max memory' in result
     os.remove(OUTPUT_FILE)
 
 
-# def test_try_free_invalid_misaligned(start_binary):
+def test_try_free_invalid_misaligned(start_binary):
```

Comment:
Honestly, I am not sure. @GrosQuildu do you know why is it commented out? Is it purely for optimization reasons, or, just a mistake?

https://github.com/pwndbg/pwndbg/commit/e3b910c5d5286ef98685932674f291d0b60b3067

I think it can probably be brought back, unless @GrosQuildu has a good reason to not to do so :P